test/Microsoft.Azure.ServiceBus.UnitTests/AmqpConverterTests.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.ServiceBus.UnitTests
+{
+    using System;
+    using System.Text;
+    using Microsoft.Azure.ServiceBus.Amqp;
+    using Xunit;
+    using BrokeredMessage = Microsoft.Azure.ServiceBus.Message;
+
+    public class AmqpConverterTests
+    {
+        public class WhenConvertingBrokeredMessagesToAmqpMessages
```

Comment:
`WhenConvertingBrokeredMessagesToAmqpMessages` vs `When_converting_brokered_messages_to_amqp_messages` I'll take 2nd option any time as it's easier to read. If this was a German word, that would be a different story 😉